### PR TITLE
feat(input): add ability to custom password toggle icon

### PR DESCRIPTION
**Pull request by izcream**
This first time to make PR to this projects. Sorry if I do anything wrong.

ps. you can change props name to another that you thinks it meaningful😊

### CODE REVIEW

**Comment by DannyFeliz** [comment]
It would be cool it you could add slots for these icon 👀

**Comment by izcream** [comment]
That's on my plan about slot implementation but still newbie on this repo architecture, need a bit time to understand this😆😍

File: src/input/src/Input.tsx [hunk 1]
```diff
@@ -1107,6 +1110,16 @@ export default defineComponent({
           )}
           {!this.pair &&
             resolveWrappedSlot($slots.suffix, (children) => {
+              const passwordVisibleIcon = this.passwordVisibleIcon ? (
+                h(this.passwordVisibleIcon)
```
Comment (07akioni) [inline src/input/src/Input.tsx:1114]: I think make it a slot is a better way. For example

```
// from src/_utils
resolveSlot(this.$slots.['password-visible'], () => [<EyeIcon />])
```

Comment (07akioni) [reply src/input/src/Input.tsx:1114]: Since it's more convient to compose.

Comment (izcream) [reply src/input/src/Input.tsx]: awesome😊

**Pull request merged**